### ISSUE TITLE
Signature verification fails where gnupg succeeds

### ISSUE THREAD

**Issue by dainnilsson** [SecurityInnovation/PGPy#295]
I've run across a signature that fails to validate in PGPy, for to me an unknown reason. The same signature is correctly verified by gpg. I'm attaching two different messages (ok.txt and fail.txt) with corresponding signatures (ok.txt.asc and fail.txt.asc), as well as the public key used to verify these (key.asc).

Note that gpg says they're both valid.
This succeeds: `$ gpg --verify fail_sig.txt fail.txt`

To reproduce it:
```python
from pgpy import PGPKey, PGPSignature

key, _ = PGPKey.from_file('key.txt')

with open('ok.txt') as f:
    message_ok = f.read()
signature_ok = PGPSignature.from_file('ok_sig.txt')

print(key.verify(message_ok, signature_ok))  # <SignatureVerification(True)>

with open('fail.txt') as f:
    message_fail = f.read()
signature_fail = PGPSignature.from_file('fail_sig.txt')

print(key.verify(message_fail, signature_fail))  # <SignatureVerification(False)>
```

Any insight into why this fails in PGPy would be greatly appreciated!
I've reproduced this both using the lastest commit from master, as well as 0.5.2.

[fail.txt](https://github.com/SecurityInnovation/PGPy/files/3789880/fail.txt)
[fail_sig.txt](https://github.com/SecurityInnovation/PGPy/files/3789897/fail_sig.txt)
[ok.txt](https://github.com/SecurityInnovation/PGPy/files/3789898/ok.txt)
[ok_sig.txt](https://github.com/SecurityInnovation/PGPy/files/3789900/ok_sig.txt)
[key.txt](https://github.com/SecurityInnovation/PGPy/files/3789901/key.txt)

**Comment by dainnilsson** [comment]
Update: I did a bit more testing and realized that it seems to be an issue with encoding. To get the failing signature to verify I simply had to explicitly encode the message as utf8. Changing the last line to:
```python
print(key.verify(message_fail.encode('utf8'), signature_fail))  # Now succeeds
```

Not sure if this is intended behavior, an issue with my system locale, or an actual issue with the library.

**Comment by Commod0re** [comment]
ohhh this is probably another `latin-1` encoding bug

**Comment by Commod0re** [comment]
I will fix this tonight :sunglasses:

**Comment by Commod0re** [comment]
this should be fixed in 986f5d272289c8421bbd3ce2682c79f287240918, which will be included in the next release

**Comment by dainnilsson** [comment]
I verified that the latest commit on master indeed resolves my issue, thanks a bunch!
